README.rst

```diff
@@ -18,6 +18,30 @@ Installation
 
     conda install -c conda-forge nc_time_axis
 
+
+Example Code
+------------
+::
+
+    import nc_time_axis
+	import netcdftime
+	import matplotlib.pyplot as plt
+	import random
+	DT = [netcdftime.datetime(year=2017, month=2, day=day) for day in range(1, 31)]
+	CDT = [nc_time_axis.CalendarDateTime(item, "360_day") for item in DT]
+	TempList = []
+	for item in range(30):
+		DayTemp = round(random.uniform(0, 12), 3)
+		TempList.append(DayTemp)
```

Comment:
This is another example of where you can use list comprehension. Also it would be a good idea to ensure the number of temperature values matches the number of calendar datetimes.

So you could rewrite lines 32-35 as:
```
temperatures = [round(random.uniform(0, 12), 3) for _ in range(len(CDT)]
```